pino.js

```diff
@@ -117,7 +119,7 @@ Object.defineProperty(Pino.prototype, 'levelVal', {
     return this._levelVal
   },
   set: function setLevelVal (num) {
-    if (typeof num === 'string') { return this._setLevel(num) }
+    if (typeof num !== 'number') { return this._setLevel(num) }
```

Comment:
This was here to catch assignments like `logger.level = 'info'` or `logger.level = '30'`. Why have you changed this? Is it integral to the new feature?